api/server/server.go

```diff
@@ -83,7 +84,7 @@ func NewServer(conf Config) (*Server, error) {
 	}
 	dm := deals.New(txndstr.Wrap(ds, "dealmodule"), c)
 
-	ip2l := ip2location.New([]string{"ip2location-ip4.bin"}) // ToDo: Flags or embed
+	ip2l := ip2location.New([]string{"./ip2location-ip4.bin"})
```

Comment:
The _ip2location-ip4.bin_ file is `COPY`ed in the Dockerfile in the same directory that the entroypoint.